.github/workflows/docker-builder-python-precommit.yml

```diff
@@ -51,4 +51,3 @@ jobs:
           IMAGE_STREAM: ${{ env.IMAGE_STREAM }}
           IMAGE_TAGS: ${{ env.IMAGE_TAGS }}
           DOCKERFILE: ${{ env.DOCKERFILE }}
-          PUSH_TO_REGISTRY: ${{ github.event_name == 'push' && github.ref_name == 'master' && github.token != null }}
```

Comment:
It seems this parameter is required, see [here](https://github.com/magma/magma/blob/master/.github/workflows/composite/docker-builder/action.yml#L31), so I guess it should just be set to false.

Reply:
Thanks, done!